[DD-169] Move _cci1 files update pronto, config.yml (#5965)
circleci/circleci-docs: jekyll/_api/v1-reference.md

<h2 id="build">Single Job</h2>

> <span class='label label-info'>Note:</span> This is also the payload for the [notification webhooks]( {{ site.baseurl }}/2.0/configuration-reference/#notify), in which case this object is the value to a key named 'payload'. 

Author: I can not see the trailing space?

> <span class='label label-info'>Note:</span> This is also the payload for the [notification webhooks]( {{ site.baseurl }}/2.0/configuration-reference/#notify), in which case this object is the value to a key named 'payload'. 

Reviewer: I don't know what the bot is picking up, but you can ignore it, I think.

> <span class='label label-info'>Note:</span> This is also the payload for the [notification webhooks]( {{ site.baseurl }}/2.0/configuration-reference/#notify), in which case this object is the value to a key named 'payload'. 

Reviewer 2: there's a space after the full stop I think

> <span class='label label-info'>Note:</span> This is also the payload for the [notification webhooks]( {{ site.baseurl }}/2.0/configuration-reference/#notify), in which case this object is the value to a key named 'payload'. 

Reviewer 2: ```suggestion
<span class='label label-info'>Note:</span> This is also the payload for the [notification webhooks]( {{ site.baseurl }}/2.0/configuration-reference/#notify), in which case this object is the value to a key named 'payload'.
```

> <span class='label label-info'>Note:</span> This is also the payload for the [notification webhooks]( {{ site.baseurl }}/2.0/configuration-reference/#notify), in which case this object is the value to a key named 'payload'. 

Reviewer 2: not that it actually matters!

<h2 id="new-build-branch">Trigger a new Build with a Branch</h2>

> <span class='label label-info'>Note:</span> For more information about build parameters, refer to the [using running tests in parrallel]( {{ site.baseurl }}/2.0/parallelism-faster-jobs/) and [optional 2.0 build parameters]({{ site.baseurl }}/2.0/env-vars/#injecting-environment-variables-with-api-v1) pages. The response for "failed" should be a boolean `true` or `null`.

Reviewer 2: ```suggestion
<span class='label label-info'>Note:</span> For more information about build parameters, refer to the [Running Tests in Parallel]( {{ site.baseurl }}/2.0/parallelism-faster-jobs/) and [optional build parameters]({{ site.baseurl }}/2.0/env-vars/#injecting-environment-variables-with-api-v1) pages. The response for "failed" should be a boolean `true` or `null`.
```

{{ site.data.api.test_metadata | api_endpoint }}

> <span class='label label-info'>Note:</span> [Learn how to set up your 2.0 builds to collect test metadata]( {{ site.baseurl }}/2.0/collect-test-data/)

Reviewer 2: ```suggestion
<span class='label label-info'>Note:</span> [Learn how to set up your builds to collect test metadata]( {{ site.baseurl }}/2.0/collect-test-data/)
```

Reviewer 2: Should be pipelines rather than builds here but you can leave for now and I will make a separate docs team ticket.
